README.md

```diff
@@ -1,60 +1,69 @@
-Jenkins Credentials Binding Plugin
-=====================
-
-Read more: [https://wiki.jenkins-ci.org/display/JENKINS/Credentials+Binding+Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Credentials+Binding+Plugin)
-
-Development
-===========
-
-Start the local Jenkins instance:
-
-    mvn hpi:run
-
-
-How to install
---------------
-
-Run
-
-	mvn clean package
-
-to create the plugin .hpi file.
-
+# Jenkins Credentials Binding Plugin
+Allows credentials to be bound to environment variables for use from
+miscellaneous build steps.
+
+Older versions of this plugin may not be safe to use. Please review the
+following warnings before using an older version:
+
+-   [Improper masking of secrets in rare
+    circumstances](https://jenkins.io/security/advisory/2018-02-05/)
```

Comment:
Delete. These lines are injected by a Confluence macro and are redundant here; the plugin site automatically shows them. Again, buggy exporter.